hack/presubmit.sh

```diff
@@ -91,12 +91,9 @@ update_crdgen() {
   CRD_YAMLS=$(find ${CRD_DIR} -iname '*.yaml' | sort)
   for i in $CRD_YAMLS; do
     sed -i '0,/---/{/---/d}' $i
-    # Currently controller-gen regenerates the status section of the CRD, which is
-    # not ideal.
-    # There is an open issue: https://github.com/kubernetes-sigs/controller-tools/issues/456.
-    # Until then, we manually delete the status field in the generated CRD yamls here.
-    sed -i '/^status:*/,$d' $i
-    echo "$(cat ${SCRIPT_ROOT}/hack/boilerplate.txt)$(cat $i)" > $i
+    # removed the crd status hack , see https://github.com/kubernetes-sigs/controller-tools/pull/630
+    echo "$(cat $i)" > $i
+    echo -e "$(cat ${SCRIPT_ROOT}/hack/boilerplate.txt)\n$(cat $i)" > $i
```

Comment:
Why have this code at all anymore?

Reply:
Yea - what happens if you remove this for loop altogether?

Reply:
if er remove the for loop we loose the LICENSE header on the yamls 
https://gist.github.com/ajayk/ffea89d2d25f61d8668984e597c3223d

@TheSpiritXIII , @pintohutch

Reply:
Ah yea that's right. We want the license headers on the individual files as well as the final combined result. Thanks for confirming!